ui/cryptomaterial/dropdown.go

```diff
@@ -152,12 +156,18 @@ func (d *DropDown) layoutActiveIcon(gtx layout.Context, index int) D {
 }
 
 func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
-	item := d.items[itemIndex]
-	radius := Radius(0)
-	clickable := item.clickable
-	if !d.isOpen {
-		radius = Radius(8)
-		clickable = d.clickable
+	var item DropDownItem
+	if itemIndex > -1 {
+		item = d.items[itemIndex]
+	}
+
+	radius := Radius(8)
+	clickable := d.clickable
+	if d.isOpen {
+		radius = Radius(0)
+		if item.clickable != nil {
```

Comment:
All Possible areas where items could panic was addressed.

Reply:
Are you basically saying that `item` cannot be nil if `d.isOpen`?
Because in line 159-162 above, we only give `item` a value if `itemIndex > -1`. So if itemIndex is -1, item will be nil.

And this point where you do `if item.clickable != nil` will panic. Unless `d.isOpen` is only true if `itemIndex > -1`.

Reply:
> All Possible areas where items could panic was addressed.

I don't see where this is done. Point me to it.